config_defaults/control.ini

```diff
@@ -78,6 +78,10 @@ key_match = docker-autotest
 product = Red Hat Enterprise Linux 7
 component = docker, docker-distribution, docker-registry, atomic,
             rhel-server-atomic, rhel-server-docker, rhel-tools-docker,
+<<<<<<< Updated upstream
```

Comment:
Eek! These are conflict markers. They indicate that you tried to `git merge` at some point, and git found changes upstream that were incompatible with changes that you made.

@cevich there are git hooks for preventing commits with conflict markers. Do you know if/how to enable them in github?